packages/js/src/components/modals/WordProofOauthSuccess.js

```diff
@@ -1,44 +1,74 @@
 /* External dependencies */
 import { __, sprintf } from "@wordpress/i18n";
+import { createInterpolateElement } from "@wordpress/element";
 
 /* Yoast dependencies */
 import { ReactComponent as Image } from "../../../images/succes_marieke_bubble_optm.svg";
 import { getWordProofSdkData } from "../../helpers/wordproof";
+import { NewButton as Button } from "@yoast/components";
+import PropTypes from "prop-types";
 
 /**
  * Creates the content for the WordProof oauth success modal.
  *
- * @returns {wp.Element} The WordProof oauth success modal.
+ * @param {object} props Functional Component props.
+ *
+ * @returns {JSX.Element} The WordProof oauth success modal.
  */
-const WordProofOauthSuccess = () => {
+const WordProofOauthSuccess = ( props ) => {
+	const {
+		closeModal,
+	} = props;
+
 	return (
 		<>
-			<div
-				style={ {
-					display: "flex",
-					justifyContent: "center",
-					marginBlock: "40px",
-				} }
-			>
-				<Image style={ { width: "175px" } } />
+			<div className={ "wordproof-modal-image" }>
+				<Image class={ "wordproof-modal-svg__success" } />
 			</div>
 
 			<p>
 				{ sprintf(
-				/* Translators: %s expands to WordProof */
-					__( "You have successfully connected to %s!",
+					/* Translators: %s expands to WordProof */
+					__( "Your page is now protected via the blockchain!!",
 						"wordpress-seo" ),
 					"WordProof"
 				) }
 				<br />
-				{ sprintf(
-				/* Translators: %s translates to the Post type in singular form */
-					__( "This %s will be timestamped as soon as you publish or update it.",
-						"wordpress-seo" ), getWordProofSdkData( "current_post_type" )
-				) }
+
+				{
+					createInterpolateElement(
+						sprintf(
+							__(
+								/** translators:
+								 * %1$s and %2$s are replaced by opening and closing <b> tags.
+								 * %3$s translates to the Post type in singular form.
+								 */
+								"The %3$s will automatically be timestamped every time you %1$supdate%2$s or %1$spublish%2$s.",
+								"wordpress-seo"
+							),
+							"<b>",
+							"</b>",
```

Comment:
More of a semantic question here. Why use a `b` tag? Isn't it meant as an emphasis for the user? (i.e. `em`?)

Reply:
Designers are our bosses :-)